cloud/docker/docker.py

```diff
@@ -27,7 +27,9 @@
 version_added: "1.4"
 short_description: manage docker containers
 description:
-  - Manage the life cycle of docker containers.
+  - This is the original Ansible module for managing the Docker container life cycle.
+  - NOTE: Additional and newer modules are available. For the latest on orchestrating containers with Ansible
+    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docsite/rst/guide_docker.rst).
```

Comment:
The U() should be parsed by our module documentation formatter code... it's part of the multiple parsers thing.   First python parses it as a string.  Then we load the docstring using yaml.  Then we find certain constructs like U(link)  and C(code_formatting) and substitute rst formatting for them. Finally, we hand it off to sphinx to format the rst into html.